data-prepper-plugins/aws-plugin/src/test/resources/test-aws-secret-manager-configuration-with-sts.yaml

```diff
@@ -1,3 +1,4 @@
 secret_id: test-secret
 region: us-east-1
-sts_role_arn: arn:aws:iam::123456789012:role/test-role
\ No newline at end of file
+sts_role_arn: arn:aws:iam::123456789012:role/test-role
+refresh_interval: null
```

Comment:
I don't think we should do this with an explicit null. It could be very confusing to users as we'd have two different meanings. 

An explicit `disable` would be clearer:

```
disable_refresh: true
```